pkgs/by-name/li/libparse-python/package.nix

```
@@ -0,0 +1,57 @@
+{
+  pkgs,
+  callPackage,
+  fetchFromGitHub,
+}:
+
+let
+  pname = "libparse-python";
+  version = "unstable-2025-12-15";
```

Version should be prefixed by `0` since it doesn't have any release tags. Also, the date should match the commit.


```suggestion
  version = "0-unstable-2025-08-30";
```